documentation/src/docs/security/securing-distributed-mode-cluster.md

```diff
@@ -310,106 +103,134 @@ Controller services can be configured in three different ways:
       ...          
    ```
 
-The following table lists the Controller's TLS and auth parameters and representative values, for quick reference. For a detailed description of these parameters, refer to [this](https://github.com/pravega/pravega/blob/master/documentation/src/docs/security/pravega-security-configurations.md) document.
+The following table lists the Controller service's TLS and auth parameters as well as samples values, for quick reference. 
+For a detailed description of these parameters, refer to 
+[this](https://github.com/pravega/pravega/blob/master/documentation/src/docs/security/pravega-security-configurations.md) document.
 
  | Configuration Parameter| Example Value |
  |:-----------------------:|:-------------|
- | `controller.security.tls.enable` | true |
- | `controller.security.tls.server.certificate.location` | /etc/secrets/controller01.pem |
- | `controller.security.tls.server.privateKey.location` | /etc/secrets/controller01.key.pem |
- | `controller.security.tls.trustStore.location` | /etc/secrets/ca-cert |
- | `controller.security.tls.server.keyStore.location` | /etc/secrets/controller01.server.jks |
- | `controller.security.tls.server.keyStore.pwd.location` | /etc/secrets/controller01.server.jks.password <sup>1</sup> |
- | `controller.zk.connect.security.enable` | false <sup>2</sup> |
- | `controller.zk.connect.security.tls.trustStore.location` | /etc/secrets/client.truststore.jks |
- | `controller.zk.connect.security.tls.trustStore.pwd.location` | /etc/secrets/client.truststore.jks.password |
- | `controller.security.auth.enable` | true |
- | `controller.security.pwdAuthHandler.accountsDb.location` <sup>3</sup> | /etc/secrets/password-auth-handler.inputfile |
- | `controller.security.auth.delegationToken.signingKey.basis | a-secret-value |
+ | `controller.security.tls.enable` | `true` |
+ | `controller.security.tls.server.certificate.location` | `/etc/secrets/controller01-server-cert.crt` |
+ | `controller.security.tls.server.privateKey.location` | `/etc/secrets/controller01-server-key.key` |
+ | `controller.security.tls.trustStore.location` | `/etc/secrets/ca-cert.crt` |
+ | `controller.security.tls.server.keyStore.location` | `/etc/secrets/controller01-server.keystore.jks` |
+ | `controller.security.tls.server.keyStore.pwd.location` | `/etc/secrets/controller01-server.keystore.jks.password` <sup>1</sup> |
```

Comment:
Use filenames here that match `generating-tls-artifacts.md` for consistency?

Reply:
Done.